tests/test.nf.test

```diff
@@ -53,7 +53,16 @@ nextflow_pipeline {
                         path("$outputDir/malt/malt_index/taxonomy.tre"),
                         "malt_index/table0.db - exists: ${path("$outputDir/malt/malt_index/table0.db").exists()}",
                         "malt_index/table0.idx - exists: ${path("$outputDir/malt/malt_index/table0.idx").exists()}",
-                        "downstream_samplesheets/databases-taxprofiler.csv - nr. lines: ${path("$outputDir/downstream_samplesheets/databases-taxprofiler.csv").readLines().size() == 8}",
+                        path("$outputDir/kmcp/database-kmcp-index/R001/_block001.uniki").exists(),
+                        "kmcp index file- minimum file size: ${file("$outputDir/kmcp/database-kmcp-index/R001/_block001.uniki").length() >= 20810000}",
+                        path("$outputDir/kmcp/database-kmcp-index/R001/__db.yml").exists(),
+                        "kmcp yml file - contains string: ${path("$outputDir/kmcp/database-kmcp-index/R001/__db.yml").readLines().any{ it.contains('alias: database-kmcp-index')}}",
+                        path("$outputDir/kmcp/database-kmcp-index/R001/__name_mapping.tsv").exists(),
+                        "kmcp tsv file - contains string: ${path("$outputDir/kmcp/database-kmcp-index/R001/__name_mapping.tsv").readLines().any{ it.contains('')}}",
```

Comment:
Checking that it contains an empty string, is that correct? You already checkl for existence above you don't need both `exists()` and checking for a string....